src/open-uri.c

```diff
@@ -668,8 +668,11 @@ handle_open_in_thread_func (GTask *task,
 
       if (open_dir)
         {
-          char *dir = g_path_get_dirname (path);
+          // Get the real path so the file manager opens the real dir and not the document portal dir.
```

Comment:
We don't use C++ style comments. This comment shouldn't be necessary if the name of the function was clearer.